rita_client/src/rita_loop/mod.rs

```diff
@@ -161,17 +83,36 @@ pub fn start_rita_loop() {
                 runner.block_on(async move {
                     manage_gateway();
 
+                    let exit_dest_price = get_exit_dest_price();
+                    let tunnels = tm_get_tunnels().unwrap();
+
+                    lcm_watch(Watch {
+                        tunnels,
+                        exit_dest_price,
+                    });
+
                     check_for_gateway_client_billing_corner_case();
                     // update the client exit manager, which handles exit registrations
                     // and manages the exit state machine in general. This includes
                     // updates to the local ip and description from the exit side
                     exit_manager_tick().await;
                     // sends an operator payment if enough time has elapsed
                     tick_operator_payments().await;
+
+                    if metrics_permitted() {
+                        send_udp_heartbeat();
```

Comment:
I don't see any migration to AWC for heartbeats here, so I don't believe this will work. Heartbeats will simply not be sent.